### DISCUSSION TITLE
Trojan Wacatac.B!ml in updater?

### DISCUSSION THREAD

**Discussion by MorgieeMorgz** [Rem0o/FanControl.Releases#721] [General]
just updated the software via the prompt when opening the software, windows defender found  Trojan:Script/Wacatac.B!ml in \FanControl\Updater.exe anyone else have this issue?
![trojan](https://user-images.githubusercontent.com/52660301/166129965-fe93d2d0-949f-4ab1-a4a2-e0e7615e6e8d.PNG)

**Comment by NZ-Wanderer** [comment]
> just updated the software via the prompt when opening the software, windows defender found Trojan:Script/Wacatac.B!ml in \FanControl\Updater.exe anyone else have this issue?

Might pay to check this out - https://github.com/Rem0o/FanControl.Releases/issues/719

**Reply by Rem0o** [reply]
See https://github.com/Rem0o/FanControl.Releases/issues/719
